index.js

```diff
@@ -36,7 +36,7 @@ const getStreamContents = async (stream, {convertChunk, getContents}, {maxBuffer
 			const chunkType = getChunkType(chunk);
 			const convertedChunk = convertChunk[chunkType](chunk, textDecoder);
 			chunks.push(convertedChunk);
-			length += convertedChunk.length;
+			length += getSize(convertedChunk);
```

Comment:
The `getSize()` addition is coming from #86 (not merged yet), to avoid merge conflicts.